mews-operations/reservations.md

```diff
@@ -288,12 +322,29 @@ The third `reservation` definition shows the partial cancellation - cancelling t
 | `12` | China Unionpay |
 | `13` | Maestro |
 
-#### Channel
+#### Source
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `code` | `int` | required | [Channel](#channels) code. |
-| `name` | `string` | required | Name of the Channel. |
+| `code` | `int` | required \(exc. new unknown sources\) | [Source](#source) code. |
+| `name` | `string` | required | Name of the [Source](#source). |
+| `type` | `int` | required | [Source Type](#source-types) code of the [Source](#source). |
+| `isPrimary` | `bool` | required | Mark the primary source where the reservation group comes from. |
```

Comment:
I would say "Indicates which source is the primary source for the reservation group"